### PR TITLE
Initial commit for the AWS ElasticSearch Service module

### PR DESCRIPTION
**Pull request by mmochan**
**Issue Type:**

New Module Pull Request

**Plugin Name:**

elastic_search

**Ansible Version:**

```
ansible 2.0.0.2 
```

**Summary:**

This module allows you to create modify and delete AWS ElasticSearch clusters.

**Example output:**

```
PLAY [Elastic Search cluster Test] *********************************************

TASK [Create Elastic Search cluster] *******************************************
changed: [localhost]

TASK [debug] *******************************************************************
ok: [localhost] => {
    "msg": {
        "changed": true, 
        "results": "Creating a new Elastic Search Cluster"
    }
}

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=0  
```

### CODE REVIEW

**Comment by robynbergeron** [comment]
Thanks @mmochan for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

**Comment by gregdek** [comment]
Thanks @mmochan for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@mmochan A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

File: cloud/amazon/elastic_search.py [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+  master_count:
+    description:
+      - Total number of dedicated master nodes, active and on standby.
+    required: false
+  zone_aware:
+    description:
+      - A boolean value to indicate whether zone awareness is enabled.
+      - If zone awareness is enabled, instance count should be at least 2
+    required: false
+  esb_options:
```
Comment (dirn) [inline cloud/amazon/elastic_search.py:55]: Should this (and all other related lines) be changed from `esb_*` to `ebs_*`?

**Comment by gregdek** [comment]
@mmochan Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@mmochan A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@mmochan Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@mmochan A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@mmochan Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Closing due to contributor inactivity.

**Pull request closed without merging**